### PR TITLE
Use notification channel on Android

### PR DESCRIPTION
**Pull request by akalin-keybase**
This is required for Android Oreo.

Hard-code channel parameters for now.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
Compare to https://github.com/zo0r/react-native-push-notification/pull/657

File: android/build.gradle [hunk 1]
```diff
@@ -16,12 +16,12 @@ allprojects {
...
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.3"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 28
```
Comment (akalin-keybase) [inline android/build.gradle:24]: Had to bump this to 28 to get it to compile

**Review by chrisnojima — COMMENTED** [review]
This is on top of the version we currently use right

File: android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java [hunk 2]
```diff
@@ -158,7 +158,7 @@ public void sendToNotificationCentre(Bundle bundle) {
                 title = context.getPackageManager().getApplicationLabel(appInfo).toString();
             }
 
-            NotificationCompat.Builder notification = new NotificationCompat.Builder(context)
+            NotificationCompat.Builder notification = new NotificationCompat.Builder(context, RNPushNotification.CHANNEL_ID)
```
Comment (akalin-keybase) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java:161]: This is called by `RNPushNotificationListenerService.handleRemotePushNotification`, which AFAICT is what is called when a GCM push notification is received.

Tested with plaintext notifications both off and on.

**Comment by akalin-keybase** [comment]
Yes, this is on top of keybase-fixes-off-302

**Review by chrisnojima — APPROVED** [review]

**Pull request merged**